charts/jindofsx/templates/fuse/daemonset.yaml

```diff
@@ -159,6 +159,12 @@ spec:
               mountPath: /token
               readOnly: true
           {{- end }}
+          {{- if .Values.ufsVolumes }}
```

Comment:
No need to mount pvc to fuse daemonset. We only need mount it to master and worker.

Reply:
fuse also need to access to pvc as client